pages/overview/what-is-jsonschema.md

```diff
@@ -2,219 +2,51 @@
-
-By using JSON Schema, we can easily define and enforce constraints, making the validation process more robust and manageable. It improves the readability of the code and reduces the chances of data-related issues.
-
-
-## Why developers use JSON Schema
-
-With JSON Schema you can:
-
-* **Describe existing data formats**: JSON Schema allows you to describe the structure, constraints, and data types of your existing JSON data formats. 
-* **Define rules and constraints**: When your JSON documents adhere to these constraints, it becomes easier to exchange structured data between applications because the data follows a consistent pattern.
-* **Clear and readable documentation**: JSON Schema supports the creation of documentation that is easily understandable by both humans and machines.
-* **Highly extensible** and can be tailored to fit your needs.
-    * You can create *custom keywords*, *formats*, and *validation rules* to suit your own requirements.
-* **Validate your data**, which helps you:
-    * **Automate testing**: JSON Schema validation enables automated testing, ensuring that data consistently adheres to the specified rules and constraints.
-    * **Enhance data quality**: By enforcing validation rules, JSON Schema helps ensure the quality of client-submitted data, preventing inconsistencies, errors, and malicious inputs.
-* **Wide range of tools availability**: The JSON Schema community has a wealth of tools and resources available across many programming languages to help you create, validate, and integrate your schemas.
-
-
-## Why organizations adopt JSON Schema
-
-* **Streamline testing and validation**: Simplify your validation logic to reduce your code’s complexity and save time on development. Define constraints for your data structures to catch and prevent errors, inconsistencies, and invalid data.
-* **Exchange data seamlessly**: Establish a common language for data exchange, no matter the scale or complexity of your project. Define precise validation rules for your data structures to create shared understanding and increase interoperability across different systems and platforms.
+
+## Benefits of JSON Schema
+### For Developers
+JSON Schema enables developers to:
+
+
+* **Describe existing data formats**: JSON Schema allows you to describe your existing JSON data formats' structure, constraints, and data types.
+* **Define rules and constraints**: When your JSON documents adhere to these constraints, It becomes easier to exchange structured data between applications because it follows a consistent pattern.
+* **Clear documentation**: JSON Schema supports the creation of documentation that is easily understandable by humans and machines.
```

Comment:
```suggestion
* **Produce clear documentation**: JSON Schema supports the creation of documentation that is easily understandable by humans and machines.
```

Reply:
We need to change the titles to match the title of the group "JSON Schema enables developers to .. "